yt/utilities/parallel_tools/tests/test_dask_helper.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import pytest
+
+from yt.config import ytcfg
+from yt.testing import assert_array_equal, assert_equal
+from yt.utilities.on_demand_imports import _dask as dask
+from yt.utilities.parallel_tools import dask_helper
+
+
+@pytest.fixture(autouse=True, scope="module")
+def with_dask_enabled():
+    # Fixture to set dask_enabled config to standard value then reset.
+    # ensures that these tests will work even if default value for dask_enabled
+    # changes while ensuring the config value gets reset to the default value
+    # after a test runs.
```

Comment:
The function name seems at odd with the comment (and the implementation), shouldn't it be `with_dask_disabled`/`without_dask_enabled`/`without_dask` ?

Reply:
oh, ya, I'll need to double check... In the tests I'm actually changing the `dask_enabled` value as needed and just using this decorator to make sure the config value is returned to its original state. So let me look again at the tests. but good catch!